kong/constants.lua

```diff
@@ -1,3 +1,9 @@
+local bit           = require "bit"
+
+
+local bor           = bit.bor
```

Comment:
ditto, these 2 instructions are adding quite a lot of whitespaces and blank lines